curp/master.go

```diff
@@ -0,0 +1,43 @@
+package curp
+
+import (
+	"net/rpc"
+	. "rtclbedit/shared"
+)
+
+type Master struct {
+	name            string
+	unsynced        []Operation
+	witness_clients map[string]*rpc.Client
+	backup_clients  map[string]*rpc.Client
+}
+
+/**
+ * RPC types
+ */
+type ExecuteArgs struct {
+}
+type ExecuteReply struct {
+}
+type SyncArgs struct {
+}
+type SyncReply struct {
+}
```

Comment:
Can we rename these structs? RN they sound like functions, which lwk confused me for a bit
```suggestion
type ExecuteArgsData struct {
}
type ExecuteReplyData struct {
}
type SyncArgsData struct {
}
type SyncReplyData struct {
}
```

Reply:
These is go RPC shenanigans, every RPC call need to pass in a "condensed" <FuncName>args, <FuncName>reply